spec/features/scheduling/scheduled_publishing_failed_spec.rb

```diff
@@ -58,13 +58,13 @@ def then_the_editors_are_notified_of_a_publishing_failure
     expected_subject = I18n.t("scheduled_publish_mailer.failure_email.subject",
                               title: @edition.title)
 
-    expect(message.to).to include(current_user.email)
+    expect(message.to).to have_content(current_user.email)
```

Comment:
This seems a bit of a strange place to use have_content since this is a plain string rather than HTML

Reply:
Good point, changed back!